target/linux/ath79/dts/qca9531_glinet_gl-xe300-nor-nand.dts

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "glinet,gl-xe300-nor-nand", "qca,qca9531";
+	model = "GL.iNet GL-XE300 Puli";
+
+	keys {
+		compatible = "gpio-keys";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins>;
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power {
+			label = "green:power";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		lan {
+			gpios = <0x0f 0x0a 0x01>;
```

Comment:
Why suddenly different?

Reply:
Because I dumped the DTS file from the original firmware and the 'dtc' command does not translate these hexadecimal values.

I will fix that, however, I just see that two leds (power and wlan) have the same second value (12). I will find the correct value for each of them next monday, as the equipment is not at home.

Reply:
Ok, the power led is not present in the original DTS, it seems it could not be handled by software.